libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
A quick experiment suggests that resetting a node makes it leave the cluster permanently. I'd still rename the function.